src/arch/riscv/isa/decoder.isa

```diff
@@ -2380,6 +2380,26 @@ decode QUADRANT default Unknown::unknown() {
                             fd = f32(defaultNaNF32UI);
                         Fd_bits = freg(fd).v;
                         }}, FloatCmpOp);
+                    0x2: fminm_s({{
+                        float32_t fs1 = f32(freg(Fs1_bits));
+                        float32_t fs2 = f32(freg(Fs2_bits));
+                        bool less = f32_lt_quiet(fs1, fs2) ||
+                                    (f32_eq(fs2, fs1) && (fs1.v & F32_SIGN));
+                        if (isNaNF32UI(fs1.v) || isNaNF32UI(fs2.v))
```

Comment:
Nit: the code in the if-else statement should be wrapped around by the curly brackets.